plivo/resources/campaign.py

```diff
@@ -123,6 +123,15 @@ def create(self,
         return self.client.request('POST', ('10dlc', 'Campaign'),
                                    to_param_dict(self.create, locals()))
 
+    @validate_args(
+            campaign_id=[required(of_type(six.text_type))],
+            campaign_alias=[required(of_type(six.text_type))],
+            method=[optional(of_type(six.text_type))],
+            url=[optional(of_type(six.text_type))],
+    )
+    def importcampaign(self,campaign_id, campaign_alias, url='', method=''):
```

Comment:
please follow the existing naming conventions, Why url and method empty string?

Reply:
@narayana-plivo do we need any default value for url and method?